demo/webpackDl/es6/src/lib/WebFS.js

```diff
@@ -0,0 +1,88 @@
+import request from 'request';
```

Comment:
I think so. It depends on node's http and https modules which webpack includes. There may be other problems with it though.